### PR TITLE
Initial commit to upload UI coverage to Codecov

### PR DESCRIPTION
**Pull request by nand-porko**
Fixes #
Add Coverage for UI #1963

### CODE REVIEW

**Comment by kannon92** [comment]
I’d also expect to see a GitHub workflow modified that uploads the coverage files to codecov.  Could you add this to our coverage action

**Comment by kannon92** [comment]
> I’d also expect to see a GitHub workflow modified that uploads the coverage files to codecov. Could you add this to our coverage action

On second thought, I think we want to only do go coverage in CodeCov at the moment.  I will take a look at this PR next week!

Thank you for your contribution!

**Comment by kannon92** [comment]
Is this ready for review?

**Comment by nand-porko** [comment]
My initial commit itself contains the code to add it to the coverage action. But I'm not sure how to test this. Can you tell me how to test it?

**Comment by kannon92** [comment]
> My initial commit itself contains the code to add it to the coverage action. But I'm not sure how to test this. Can you tell me how to test it?

Your commit runs the coverage via `npm run test` and I think that is sufficient for this.  I realize that we don't want to push coverage on the entire repo at the moment.  

Do you see a coverage file when you run `npm run test` and does it look sane?  If it does then I think that is good.

File: internal/lookout/ui/jest.config.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// Sync object
```
Comment (kannon92) [inline internal/lookout/ui/jest.config.js:1]: https://create-react-app.dev/docs/running-tests/#coverage-reporting

Reading this I don't know if we need a special Jest config for this.  Could you remove and see if you can just add --test?

Comment (nand-porko) [reply internal/lookout/ui/jest.config.js:1]: My bad, we don't need the jest.config.js file. I have changed the jest config in package.json to spit out coverage report in cobertura (as required by Codecov).

**Comment by nand-porko** [comment]
Yes, the coverage file looks sane!

File: internal/lookout/ui/jest.config.js (hunk 1 above)
Comment (kannon92) [reply internal/lookout/ui/jest.config.js:1]: Once you remove this I think it is good to go!

File: internal/lookout/ui/package-lock.json [hunk 2]
```diff
@@ -0,0 +1,15355 @@
+{
```
Comment (kannon92) [inline internal/lookout/ui/package-lock.json:1]: Actually can you remove this

File: internal/lookout/ui/jest.config.js (hunk 1 above)
Comment (kannon92) [reply internal/lookout/ui/jest.config.js:1]: Please remove Jest config and the package-lock.json.  We use yarn now so I forgot that shouldn't be there.

**Pull request closed without merging**